### PR TITLE
now if a user sigend in , he remain signed in until he log out

### LINKED ISSUE #15: Splash screen for already logged-in users

### PR DESCRIPTION
**Pull request by haitham-kablan**
Fix #15.

### CODE REVIEW

**Comment by haitham-kablan** [comment]
app now take like 3-4 seconds to show the main page , becuase when we open the app we ask the database if therer is a user logged in or not , it is better to add liltle loading widget before the app lunches instaed of white screen

File: hadar/lib/main.dart [hunk 1]
```diff
@@ -18,12 +19,13 @@ void main() async {
...
   await Firebase.initializeApp();
   const double marginSize = kIsWeb ? 0.0 : 0.0;
+  Widget logged_in_user = await CurrentUser.init_user();
   runApp(MaterialApp(
     debugShowCheckedModeBanner: false,
     home: Container(
         margin: const EdgeInsets.only(left: marginSize, right: marginSize),
         child: Scaffold(
-          body: LogInPage(),
+          body: logged_in_user != null ? logged_in_user : LogInPage(),
```
Comment (elazarg) [inline hadar/lib/main.dart:28]: Concisely: `logged_in_user ?? LogInPage()`

Comment (haitham-kablan) [reply hadar/lib/main.dart:28]: I will change it , and close the pull request.
thank you!

Comment (haitham-kablan) [reply hadar/lib/main.dart:28]: I wont close it , I have to add more the loading screen. for better user experience.
thank you anyway!

Comment (elazarg) [reply hadar/lib/main.dart:28]: There's no need to close anything. Simply make the change and push.

**Comment by haitham-kablan** [comment]
Fixed it , now I just need to upload good temporary picture to the loading screen before the app lunches

**Pull request merged**